server/src/test/scala/services/crunch/workload/WorkloadSpec.scala

```diff
@@ -52,7 +61,14 @@ class WorkloadSpec extends Specification {
     val procTimes = Map(PaxTypeAndQueue(PaxTypes.EeaMachineReadable, Queues.EeaDesk) -> 1.5)
 
     val flightSplitMinutes = WorkloadCalculator
-      .flightToFlightSplitMinutes(ApiFlightWithSplits(arrival, splits, None), procTimes, Map(), false)
+      .flightToFlightSplitMinutes(
+        ApiFlightWithSplits(arrival, splits, None),
+        procTimes,
+        Map(
+
+
+        )
+        , false, pcpPaxFn)
```

Comment:
Is this intellij 'chopping down'? There're 2 blank lines in the Map() for some reason